### PR TITLE
[change] Merged openwisp-radius container in dashboard and API #197

### LINKED ISSUE #197: [change] Merge openwisp-radius container in dashboard and API
Let's simplify. The API endpoints of openwisp-radius can be served by the API container, if openwisp-radius is enabled.
The dashboard container can show its admin, if enabled, this should be already working this way.
The openwisp-radius container can then be eliminated.

### PR DESCRIPTION
**Pull request by codesankalp**
- API container serves openwisp-radius API
- Dashboard container has support for openwisp radius features
- Eliminated openwisp-radius container
- Change port of websocket from 8003 to 8002 for consistency

Closes #197

### CODE REVIEW

**Review by nemesisdesign — APPROVED** [review]
With the changes done in https://github.com/openwisp/openwisp-config/pull/163/commits/2e7d1c5fb4a5744370f6247effa4b304a10f6eb8 I think now it's ready.

**Comment by nemesisdesign** [comment]
My previous review is a mistake, I thought I was replying to another PR, sorry!

File: README.md [hunk 1]
```diff
@@ -76,7 +73,7 @@ The `auto-install.sh` script can be used to quickly install a simple instance of
 
 If you have created a [`.env` file](docs/ENV.md) to configure your instance, then you can use it with the script otherwise.
 
-**It asks 6 questions for application configuration, 4 of them are domain names.** The dashboard, api, radius & openvpn can be setup on different domain, **please ensure the domains you enter point to your server**. The remaining **2 questions are email id** for site manager email (used by django to send application emails) and letsencrypt (used by [certbot](https://certbot.eff.org/) to issue https certs on this address.)
```
Comment (atb00ker) [inline README.md:79]: 3

File: deploy/auto-install.sh [hunk 2]
```diff
@@ -94,9 +94,6 @@ setup_docker_openwisp() {
         # API Domain
         echo -ne ${GRN}"(2/6) Enter API domain (blank for api.${domain}): "${NON};
```
Comment (atb00ker) [inline deploy/auto-install.sh:95]: Question's number needs to change as well.

**Review by atb00ker — COMMENTED** [review]
Wow, great work.
This needs loads of testing! 😄

**Review by nemesisdesign — APPROVED** [review]
I did some basic testing and I see the openwisp-radius API endpoints in the API container. The RADIUS module works fine in the admin, I also tried the user generation feature with PDF and CSV to ensure the PDF and CSV files get downloaded properly.

LGTM :+1:.

**Comment by atb00ker** [comment]
The image can be edited using Miro.

1. Create / login to Miro account (Free)
2. Create an empty / Open existing board
3. Go to https://miro.com/app/board/o9J_kvt3edQ=/?moveToWidget=3074457361507412175&cot=14 and copy the image
4. Edit it as you like and export the image and upload here.
(Please share the link to your changes as well, so that it can be used in the future)

**Comment by atb00ker** [comment]
Sweet, that was smooth, thanks @codesankalp ! 😄

**Review by nemesisdesign — APPROVED** [review]

**Pull request merged**